Specs/DateTime/German/DatePeriodParser.json

```diff
@@ -925,7 +925,7 @@
     "NotSupported": "javascript, python",
     "Results": [
       {
-        "Text": "zweitausender",
+        "Text": "Die zweitausender",
```

Comment:
Lowercase

Reply:
I believe it is case sensitive for English as well. Could you please check? See this for example:

 {
    "Input": "I'll be out from 2 to 7 Feb, two thousand and eighteen",
    "Context": {
      "ReferenceDateTime": "2016-11-07T00:00:00"
    },
    "NotSupported": "javascript, python",
    "Results": [
      {
        "Text": "from 2 to 7 Feb, two thousand and eighteen",
        "Type": "daterange",
        "Value": {
          "Timex": "(2018-02-02,2018-02-07,P5D)",
          "FutureResolution": {
            "startDate": "2018-02-02",
            "endDate": "2018-02-07"
          },
          "PastResolution": {
            "startDate": "2018-02-02",
            "endDate": "2018-02-07"
          }
        },
        "Start": 12,
        "Length": 42
      }
    ]
  },